### ISSUE TITLE
Web3/Parity Console does not work on Windows 7

### ISSUE THREAD

**Issue by jdetychey** [ethcore/parity#4376]
Browser : Chrome
Console browser:

```
index.ef7beb4014.js:6 [checkUpgrade] available: null version: Object consensusCapability: unknown
index.ef7beb4014.js:5 [lookupAddress] (0x7bbc4f1a27628781b96213e781a1b8eec6982c1db8fac739af6e4c5a55862c03) dappreg: 0x724A8602fc0C2b346f8eC56Df2913710742d3fD0
index.ef7beb4014.js:5 [lookupAddress] (0xf49089046f53f5d2e5f3513c1c32f5ff57d986e46309a42d2b249070e4e72c46) signaturereg: 0x336D7c77B87754fF5c5CF8138605bAAB05B1D9Ba
index.ef7beb4014.js:5 [lookupAddress] (0x0a8048117e51e964628d0f2d26342b3cd915248b59bcce2721e1d05f5cfa2208) tokenreg: 0x7Ae6d608CD96f5D34eD989211de85d7de1a31585
index.ef7beb4014.js:5 [lookupAddress] (0x058740ee9a5a3fb9f1cfa10752baec87e09cc45cd7027fd54708271aca300c75) githubhint: 0xfA29053696C1353a53656a3F15710CD59D08Fce8
index.ef7beb4014.js:5 [lookupAddress] (0xae097a8b4e6cf4fabb9d95b81633b43f0d56b70de6abe5108287e01dc6664eb7) badgereg: 0xcF5A62987294fd2087252FD812443508528C52bF
index.ef7beb4014.js:19 Error: Certifier 0 does not exist.
(anonymous) @ index.ef7beb4014.js:19
index.ef7beb4014.js:19 Error: Certifier 1 does not exist.
(anonymous) @ index.ef7beb4014.js:19
index.ef7beb4014.js:19 Error: Certifier 2 does not exist.
(anonymous) @ index.ef7beb4014.js:19
index.ef7beb4014.js:19 Error: Certifier 3 does not exist.
(anonymous) @ index.ef7beb4014.js:19
index.ef7beb4014.js:19 Error: Certifier 4 does not exist.
(anonymous) @ index.ef7beb4014.js:19
index.ef7beb4014.js:19 Error: Certifier 6 does not exist.
(anonymous) @ index.ef7beb4014.js:19
index.ef7beb4014.js:19 Error: Certifier 7 does not exist.
(anonymous) @ index.ef7beb4014.js:19
index.ef7beb4014.js:19 Error: Certifier 10 does not exist.
(anonymous) @ index.ef7beb4014.js:19
index.ef7beb4014.js:19 Error: Certifier 11 does not exist.
(anonymous) @ index.ef7beb4014.js:19
http://127.0.0.1:8180/parity-utils/parity.js Failed to load resource: the server responded with a status of 404 (NOT FOUND)
http://127.0.0.1:8180/parity-utils/web3.js Failed to load resource: the server responded with a status of 404 (NOT FOUND)
console.js:22 Uncaught TypeError: Cannot read property 'api' of undefined
    at console.js:22
http://127.0.0.1:8080/api/content/0b9b62a9262f75461191d4e8bf686c56528cbc0fe885c1f5878052ca8b2f65bf Failed to load resource: the server responded with a status of 502 (Bad Gateway)
http://127.0.0.1:8080/api/content/c3962e2eab7afaeb9cd11522381723afbafdc41dc7ba31bee472e187c4459e81 Failed to load resource: the server responded with a status of 502 (Bad Gateway)
http://127.0.0.1:8080/api/content/22b924801e1971659a51956dcdfc5a2d592d8bdd03780dd72d5bc4c84b595b4c Failed to load resource: the server responded with a status of 502 (Bad Gateway)
http://127.0.0.1:8080/api/content/cc7cc620703eb4650c0eeaaaaf83731a2838ec7e92f9dabd437f3e93f8e8ed53/ Failed to load resource: the server responded with a status of 502 (Bad Gateway)
http://127.0.0.1:8080/api/content/d40679a3a234d8421c678d64f4df3308859e8ad07ac95ce4a228aceb96955287 Failed to load resource: the server responded with a status of 502 (Bad Gateway)
http://127.0.0.1:8080/api/content/4343756dbc006463b4086710c992083871966aa28c8ce74d16c6d74d4ddb7c2f Failed to load resource: the server responded with a status of 502 (Bad Gateway)
http://127.0.0.1:8080/api/content/2534b44f685b6399bf63f86679128216c43e9a58be1dfb58533923f0bcffeba7 Failed to load resource: the server responded with a status of 502 (Bad Gateway)
sw.js:1 Service Worker termination by a timeout timer was canceled because DevTools is attached.
```

**Comment by jdetychey** [comment]
Let me know if you need more info

**Comment by jacogr** [comment]
Thank for the report.

Which version of Parity are you running?

**Comment by jdetychey** [comment]
1.5

**Comment by 3esmit** [comment]
Related to https://github.com/ethcore/parity/issues/4290

**Comment by tomusdrw** [comment]
Yup, seems like a duplicate of #4290, will be fixed in 1.5.1/1.6
